packages/core/src/docdb/explorer/dbClusterNode.ts

```diff
@@ -22,16 +24,23 @@ export class DBClusterNode extends AWSTreeNodeBase implements AWSResourceNode {
     name: string = this.cluster.DBClusterIdentifier ?? ''
     arn: string = this.cluster.DBClusterArn ?? ''
 
-    constructor(readonly cluster: DBCluster) {
+    constructor(readonly cluster: DBCluster, readonly client: DocumentDBClient) {
```

Comment:
Should the `client` parameter be used or saved somewhere?

Reply:
It's used on line 36

Reply:
Ah, I had to look up the [documentation for parameter properties](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties). I'm not super familiar with classes as I try to stay in functional land.

Looks good!